Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_System.m

```diff
@@ -469,6 +470,19 @@
     return [[NSBundle mainBundle] pathForResource:@"embedded" ofType:@"mobileprovision"] != nil;
 }
 
+/**
+ * Check if the embdded.mobileprovision provisions all devices
+ * If true the profile is for `enterprise`, if not it is `adhoc`
+ */
+static bool
+mobileProvisionProfileProvisionsAllDevices(void)
+{
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"embedded" ofType:@"mobileprovision"];
+    SentryMobileProvisionParser *parser = [[SentryMobileProvisionParser alloc] initWithPath:path];
```

Comment:
`m`: We now get the path for the `mobileprovision` in two different places. Once here and once in hasEmbeddedMobileProvision https://github.com/getsentry/sentry-cocoa/blob/34ca22f558f73d31ce49e777fa6a28c088ec1ded/Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_System.m#L467C1-L470

I think we could move the logic from `hasEmbeddedMobileProvision` into the `SentryMobileProvisionParser` to only have it in one place.